sdk/tests/conformance/extensions/webgl-compressed-texture-s3tc.html

```diff
@@ -435,7 +451,10 @@
   }
 }
 
-function testDXTTexture(test) {
+function testDXTTexture(test, useTexStorage) {
+    if (!useTexStorage)
+        return;
```

Comment:
Thank you for catching that! I'm sorry I missed it when cleaning up the CL. Yes, I had to take out most of the test's code when trying to figure out where the errors were coming from initially.